riotfile.py

```diff
@@ -2586,6 +2586,22 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 "molten": [">=0.6,<0.7", ">=0.7,<0.8", ">=1.0,<1.1", latest],
             },
         ),
+        Venv(
+            name="gunicorn",
+            command="pytest {cmdargs} tests/contrib/gunicorn",
+            pkgs={"requests": latest},
+            venvs=[
+                Venv(
+                    pys="2.7",
+                    # Gunicorn ended Python 2 support after 19.10.0
+                    pkgs={"gunicorn": "==19.10.0"},
+                ),
+                Venv(
+                    pys=select_pys(min_version="3.5"),
+                    pkgs={"gunicorn": ["==19.10.0", "==20.0.4", latest]},
```

Comment:
Do we need to test version 19 here as well? I'm thinking it might be good to skip it if we can to shorten the CI run duration. WDYT?

Reply:
I think we've written on our [documentation](https://github.com/DataDog/dd-trace-py/pull/4776/files#diff-8d068e8797e88947c320f79e856c3e16a72b730124a8f9d7031e2c4680dfa534) that we support gunicorn from version 19.10 onwards.

Considering CI for the gunicorn suite takes only 6 minutes right now, we should be fine for now but once we add more tests and it does get longer, we should definitely revisit this and see if we really need to test this far back (or if we can just test one minor version in version 20).